src/main/java/com/google/sps/servlets/InfoCardServlet.java

```diff
@@ -74,15 +95,19 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
           .println(
               "Could not find election with ID " + optionalElectionId.get() + " in Datastore.");
       response.setStatus(400);
+      return;
     }
 
+    logger.logp(Level.INFO, SOURCE_CLASS, "doPut", "Got election ID and user address.");
+
     Entity electionEntity = optionalEntity.get();
-    Election election = Election.fromEntity(electionEntity);
 
-    // Don't need to make the API call if this Election object has already been populated.
-    if (election.isPopulatedByVoterInfoQuery()) {
-      return;
-    }
+    // Temporary solution to repeated information in the Datastore.
```

Comment:
Leave a TODO for gianluca to clean this up. We talked with him about working on this as his one-pager worthy project.

Reply:
Got it, will do.